### DISCUSSION TITLE
Release v1.2.2

### DISCUSSION THREAD

**Discussion by ChenglongMa** [ChenglongMa/LOLauncher#16] [General]
# 更新日志 （非常抱歉此次频繁更新😌）

[![Downloads](https://img.shields.io/github/downloads/ChenglongMa/LOLauncher/1.2.2/total)](https://github.com/ChenglongMa/LOLauncher/releases/tag/1.2.2)

1. 🐛 **FIX**: 修复了欧洲、日本等非英语国家地区 `一键喊话` 功能无法识别游戏进程的bug；

## What's Changed
* Release v1.2.2 by @ChenglongMa in https://github.com/ChenglongMa/LOLauncher/pull/15

**Full Changelog**: https://github.com/ChenglongMa/LOLauncher/compare/1.2.1...1.2.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ChenglongMa/LOLauncher/releases/tag/1.2.2'>Release v1.2.2</a>.</em>

**Comment by Effyovo** [comment]
谢谢大佬的插件!!特别好用，已经从老版本用到现在了。
但是每次需要手动启动客户端有一丢丢麻烦：
想请问是否有办法能够一键启动客户端呢？

非常感谢

**Reply by ChenglongMa** [reply]
感谢您的支持☺️☺️！拳头为了防作弊，现在强制用户必须从他的客户端启动游戏，所以暂时没法自动启动游戏😓，非常抱歉！

**Reply by ChenglongMa** [reply]
哦哦，我想到个办法，等着我更新一下，感谢耐心等待！！

**Reply by Ptou17** [reply]
> 谢谢大佬的插件!!特别好用，已经从老版本用到现在了。 但是每次需要手动启动客户端有一丢丢麻烦： 想请问是否有办法能够一键启动客户端呢？
> 
> 非常感谢

我是在 运行里面输入shell:startup 然后直接把程序丢进去 然后就开机自启了 看看对你有没有用

**Comment by Ptou17** [comment]
突然发现跟大佬是校友 我今年刚RMIT硕士毕业哈哈哈

**Reply by ChenglongMa** [reply]
哈哈，幸会幸会！我是今年博士毕业。你参加这个月18号的毕业典礼吗？

**Reply by Ptou17** [reply]
哈哈哈恭喜马博士 我不参加了嘿嘿本科参加过了 体验一次就行了

**Reply by ChenglongMa** [reply]
哈哈，谢谢！有机会再见 🙏

**Comment by Hurrrrt** [comment]
Hello. I’m a Japanese university student studying IT and a League of Legends player. I’ve been using this app  when I play LoL, but since the UI is in Chinese, I’d like to personally change it to Japanese. However, I can’t find where the UI code files are located on my PC. If it’s not too much trouble, could you please let me know where they are?

**Reply by ChenglongMa** [reply]
Hi @Hurrrrt,

Thanks for your attention to this app. 

To change the LOL UI to Japanese, you can simply set it in the built-in LOL Client:

![Snipaste_2025-05-11_09-00-42](https://github.com/user-attachments/assets/5c525e68-3ffe-47a9-b625-8ee96d668844)

Please let me know if you need further information.

Thanks :)

Best,
Chenglong

**Reply by Hurrrrt** [reply]
Sorry for the confusion in my previous message. I didn't mean the Riot Client, but rather the LoLauncher. I would like to make the "Language Settings" UI shown in the screenshot as Japanese as possible.
![スクリーンショット 2025-05-13 121042](https://github.com/user-attachments/assets/b765ac06-ccde-4c7f-bfd6-7398936243f2)

**Reply by ChenglongMa** [reply]
Hi @Hurrrrt,

This app has the same functions as the native Riot Client when changing to other languages *except Chinese*. Therefore, if you want to use **Japanese** in the game, you can just use the Riot Client.

For your question, since I don't understand Japanese, the app's current version does not have a translation into Japanese configured. I can add this feature if you need, but I'll have to wait a while, I've been busy lately. Sorry about that.

Cheers,
Chenglong

**Reply by Hurrrrt** [reply]
Thank you for replying despite your busy schedule. I want to become an engineer in the future too, so I’ll keep working hard at my studies!

**Reply by ChenglongMa** [reply]
Hi @Hurrrrt,

I'm very happy that you are interested in engineering :). Please feel free to `fork` this project and update the code as you like. I'm also very happy to discuss programming-related questions with you :)

Thanks!
Chenglong

**Comment by JeffS-Debug** [comment]
你好大佬，我不知道现在这个程序还能不能用，我今天试了一下，发现下载完成之后启动lol会先打开我的拳头，然后我点击游玩lol之后依然是全英文的。然后我关闭lol之后拳头并没有显示更新的按钮。请问这个是怎么回事

**Reply by ChenglongMa** [reply]
你好 @JeffS-Debug，这个还能用，我天天都用，哈哈。你描述的过程是对的，是先打开拳头自己的client，然后要点击"Play"进入游戏。没更新成功可能是因为语言包没下载完全，你可以多试几次看看。如果不行再跟我说哈。

**Reply by JeffS-Debug** [reply]
感谢大佬，我重新安装了一下拳头和lol，之后能正常使用了。

**Reply by ChenglongMa** [reply]
那就好！不好意思还让你重新安装了
